lib/js/components/Image/Image.vue

```diff
@@ -0,0 +1,81 @@
+<template>
+	<div class="ds-image">
+		<img
+			class="ds-image__image"
+			:class="{
+				'-ds-contain': fit === IMAGE_FITS.CONTAIN,
+				'-ds-cover': fit === IMAGE_FITS.COVER,
+			}"
+			draggable="false"
+			loading="lazy"
+			:src="src"
+			@error="isLoading = false"
+			@load="isLoading = false"
+		/>
+		<div v-if="isLoading" class="ds-image__loader">
+			<prime-skeleton width="100%" height="100%" border-radius="0" />
+		</div>
+	</div>
+</template>
+
+<style lang="scss" scoped>
+.ds-image {
+	height: 100%;
+	position: relative;
+
+	&__image {
+		height: 100%;
+		user-select: none;
+		width: 100%;
+
+		&.-ds-contain {
+			object-fit: contain;
+		}
+
+		&.-ds-cover {
+			object-fit: cover;
+		}
+	}
+
+	&__loader {
+		bottom: 0;
+		left: 0;
+		position: absolute;
+		right: 0;
+		top: 0;
+	}
+}
+</style>
+
+<script lang="ts">
+import PrimeSkeleton from 'primevue/skeleton';
+import { defineComponent, PropType } from 'vue';
+import { IMAGE_FITS, ImageFit } from './Image.consts';
+
+export default defineComponent({
+	// <image> is "an ancient and poorly supported precursor to the <img> element"
+	// so there should be no conflicts. Also, we prefix all DS components with ds-
+	// https://developer.mozilla.org/en-US/docs/Web/HTML/Element#obsolete_and_deprecated_elements
+	// eslint-disable-next-line vue/no-reserved-component-names
+	name: 'Image',
+	components: {
+		PrimeSkeleton,
```

Comment:
@rogatty Shouldn't we create DsSkeleton and use it here instead of using `primevue/skeleton` directly?

Reply:
I don't know, it might be a question for #design-system. We need to add a new component for wnl-platform but maybe we don't need it for DS components?

Reply:
IMO, we should have a DsSkeleton component and we should use it here. If we decide to introduce some changes (eg. colors or animation), this place should also follow that change.